### PR TITLE
added Dockerfile and .dockerignore

### PR DESCRIPTION
**Pull request by danielsh28**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+build
```
Comment (yuvalbl) [inline .dockerignore:2]: please add some comments to explain it
see the following example:
https://gist.github.com/jkinkead/55db75d57df756c6d66b4c3f395df521

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM node:13.12.0-alpine
+
+WORKDIR /app/front/
+
+#install app dependencides
+COPY package*.json ./
```
Comment (yuvalbl) [inline Dockerfile:6]: copy `package-lock.json` as well
https://docs.npmjs.com/configuring-npm/package-lock-json.html

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+#add the app files itself
+COPY . ./
+#expose the app port
+EXPOSE 3000
+
+
+#start app
+CMD ["npm","start"]
+
```
Comment (yuvalbl) [inline Dockerfile:17]: npm start should only used for dev
please add a production dockerfile. see:
https://mherman.org/blog/dockerizing-a-react-app/#production

File: Dockerfile (hunk 2 above)
Comment (danielsh28) [reply Dockerfile:6]: I used a wild card * especially for that- see reference in [https://docs.docker.com/engine/reference/builder/#copy](url).
and also [https://www.robinwieruch.de/docker-create-react-app-development](url)

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**